### PR TITLE
Add new `Style/BisectedAttrAccessor` cop

### PR DESCRIPTION
**Pull request by fatkodima**
This cop checks where `attr_reader :foo` and `attr_writer :foo` can be combined into `attr_accessor :foo`.

I was surprised when I haven't found a similar check.

There were a couple of offenses in my project, that stayed after not so careful refactoring. So I would like to have a cop with suggestions.

### CODE REVIEW

File: lib/rubocop/cop/style/bisected_attr_accessor.rb [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+      #     attr_writer :bar
+      #   end
+      #
+      #   # good
+      #   class Foo
+      #     attr_accessor :bar
+      #   end
+      #
+      class BisectedAttrAccessor < Cop
+        MSG = 'Combine both accessors into `attr_accessor`.'
```
Comment (bbatsov) [inline lib/rubocop/cop/style/bisected_attr_accessor.rb:24]: I'd mention the attribute name in the message for clarity.

**Comment by bbatsov** [comment]
Great work! I'm guessing the cop should also cover the somewhat deprecated `attr`, which I think was an alias for `attr_reader`.

**Comment by fatkodima** [comment]
Updated with suggestions.

**Comment by bbatsov** [comment]
Thanks!

**Pull request merged**